usmqe_tests/api/tendrl/others/conftest.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+
+
+@pytest.fixture(
+    params=[{
+        "name": "Tom Hardy",
+        "username": "thardy",
+        "email": "thardy@tendrl.org",
+        "role": "admin",
+        "password": "pass1234",
+        "password_confirmation": "pass1234"}])
+def valid_user_import(request):
```

Comment:
Wouldn't it make more sense to name this fixture as `valid_user_data` instead? The current name sounds as if it were concerned with importing user or something like that.